src/styles/typography.css

```
@@ -28,4 +28,45 @@
+   *
+   * `.quiet-link` is the same treatment for something that behaves like a link
+   * but isn't an anchor — today the project "Read more" toggle. Shared so the
+   * two can't drift apart, which is what `.rich-links` was split out for.
+   */
+  .rich-links a,
+  .quiet-link {
+    color: inherit;
+    text-decoration: underline;
+    text-decoration-thickness: 1px;
+    text-underline-offset: 4px;
+    text-decoration-color: color-mix(in srgb, currentColor 35%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)